Libplanet/Blocks/IPreEvaluationBlock.cs

```diff
@@ -15,4 +15,8 @@ public interface IPreEvaluationBlock<T> : IBlockContent<T>, IPreEvaluationBlockH
         where T : IAction, new()
     {
     }
+
+    public interface IPreEvaluationBlock : IBlockContent, IPreEvaluationBlockHeader
+    {
+    }
 }
```

Comment:
IMO this interface need some comment. In my case, I'm confuse why two `IPreEvalutationBlock` in here. I think `IPreEvaluationBlock<T>` is legacy and we will go to `IPreEvaluationBlock` but is it right?

Reply:
> `IPreEvaluationBlock<T>` is legacy and we will go to `IPreEvaluationBlock` but is it right?

Right. 

@moreal As he said, it would be better with comments about that.

Reply:
I complemented the descriptions for `IPreEvaluationBlock` in [a68a591](https://github.com/planetarium/libplanet/pull/2539/commits/a68a591197c349b537cc75c4ffefd76382571959) and the plan in [e0c715e](https://github.com/planetarium/libplanet/pull/2539/commits/e0c715e23eb66e4386655c5269ba9d4dd199e4ad)